src/api/spec/models/cloud/backend/upload_job_spec.rb

```diff
@@ -155,6 +155,7 @@
                 <package>rpm</package>
                 <arch>x86_64</arch>
                 <filename>rpm-4.14.0-504.2.x86_64.rpm</filename>
+                <details>AMI-super04958743</details>
```

Comment:
[nitpick] This shouldn't be necessary if you don't test it, no?

Reply:
For consistency with the data?

Reply:
Ok.

But then I would use a valid ami instead -> ``ami-42``